qcodes/plots/qcmatplotlib.py

```diff
@@ -539,6 +576,11 @@ def scale_formatter(i, pos, scale):
                 # TODO: include all traces when calculating maxval etc.
                 trace = traces[0]
             for axis in 'x', 'y', 'z':
+                log_scale = False
+                if (axis in ['x', 'y'] or hasattr(subplot, 'qcodes_colorbar')) and \
+                        getattr(subplot, "get_{}scale".format(axis))() == 'log':
+                    log_scale = True
```

Comment:
I think this can be written a bit more clearly. What about first creating a variable `axis_scale`, and then setting to True / False dependent on these if statements.

Reply:
Good idea, done.